priv/catalog/205AF7.yaml

```diff
@@ -0,0 +1,51 @@
+id: "205AF7"
+name: fencing enabled
+group: Pacemaker
+description: |
+  Fencing is enabled in the cluster properties 'cib-bootstrap-options': stonith-enabled
+remediation: |
+  ## Abstract
+  Fencing is mandatory to guarantee data integrity for your SAP Applications.
+  Running a HA Cluster without fencing is not supported and might cause data loss.
+
+  ## Remediation
+  Execute the following command to enable it:
+  ```
+  crm configure property stonith-enabled=true
+  ```
+
+  ## References
+  AZURE:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker#create-a-stonith-device-on-the-pacemaker-cluster
+
+  AWS:
+
+    - https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-on-aws-cluster-resources.html
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles#configure_the_general_cluster_properties
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-cluster-bootstrap-and-more
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-fencing.html#sec-ha-fencing-recommend
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-cluster-bootstrap-and-more
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-fencing.html#sec-ha-fencing-recommend
+
+facts:
+  - name: fencing_enabled
```

Comment:
As fencing and stonith are mostly used as synonyms I will stay with 'fencing_enabled'.

Reply:
I see, however I was pointing to a different nuance :smile: 

What I was saying is that what the gatherer returns is not really a fact about fencing being enabled or not, even if we name the fact `fencing_enabled`

```yaml
facts:
  - name: fencing_enabled # this can be anything
    gatherer: cibadmin
    argument: cib.configuration.crm_config.cluster_property_set
```
what the gatherer is returning is _something from which we are able to get the information about fencing being enabled_ after we applied some extraction/transformation function like `.find(|item| item.id == "cib-bootstrap-options")].nvpair.find(|prop| prop.name == "stonith-enabled").value` to `facts.<whatever the fact name>`.

I should have been clearer. However I wouldn't bother too much for now :wink:

Reply:
:smile:  ok, now I got it.
And yes, viewed from this angle you are right. So I will take a cup of tea and try to find a better matching name.